api/models.py

```diff
@@ -194,14 +194,14 @@ class Node(DatabaseModel):
         default_factory=datetime.utcnow,
         description='Timestamp when node was last updated'
     )
-    timeout: Optional[float] = Field(
-        default=24.0,
-        description='Maximum time in hours to wait for node to get it \
-completed',
-        ge=0.0,
-        le=24.0
+    timeout: Optional[datetime] = Field(
+        description='Node expiry timestamp'
     )
 
+    def set_timeout(self):
+        """Set Node.timeout value based on Node.created timestamp"""
+        self.timeout = self.created + timedelta(hours=24)
+
```

Comment:
I'd make it more generic. I think that later we can move timeout to the configuration file, this can facilitate this.

```suggestion
    def set_timeout(self, hours: int=24):
        """Set Node.timeout value based on Node.created timestamp
        hours (int): number of hours after which the node will time out (default: 24)
        """
        self.timeout = self.created + timedelta(hours=hours)

```

Reply:
Updated.